app/__init__.py

```diff
@@ -299,6 +299,41 @@ def update_usernames():
     'failed': failed,
   }
 
+@app.route("/spec")
+def get_apispec():
+    return jsonify(spec.to_dict())
+
+@app.route('/account/credit_balance', methods=['GET'])
```

Comment:
can you delete this actually while we're doing this, it was for something else

Reply:
@mjmayank do you mean the `get_apispec` endpoint?

Reply:
no the /account/credit_balance